core/client/hdfs/src/main/java/alluxio/hadoop/FileSystem.java

```diff
@@ -43,6 +45,13 @@
  */
 @NotThreadSafe
 public class FileSystem extends AbstractFileSystem {
+  public static final String MASTER_LOGICAL_NAMESERVICES = "alluxio.master.nameservices.%s";
+  public static final String MASTER_LOGICAL_RPC_ADDRESS = "alluxio.master.rpc.address.%s.%s";
+  public static final String MASTER_LOGICAL_ZOOKEEPER_NAMESERVICES =
+      "alluxio.master.zookeeper.nameservices.%s";
+  public static final String MASTER_LOGICAL_ZOOKEEPER_ADDRESS =
+      "alluxio.master.zookeeper.address.%s.%s";
+
```

Comment:
But these are really needed in the configuration file, I think it makes sense to write them in the PropertyKey.

Reply:
`PropertyKey` should not be the place where all properties for all components are "documented". That's what documentations are for.

Reply:
@qian0817 Can you clarify with an example how these properties are to be used? I'll update the docs to reflect that afterwards.

The main question at the moment is that @rongrong has found that these properties don't get parsed from the `alluxio-site.properties`-level configuration whatsoever. Rather, they are being read out of the Hadoop config (which would be set in `hdfs-site.xml` and/or `core-site.xml`). That's why they're being removed from `PropertyKey`. Is this correct?

Reply:
@ZhuTopher  As you said, these configurations will only be read from Hadoop configuration. If they are removed from `PropertyKey`, I think a more appropriate place is needed, as they would be difficult to find if exist in the `FileSystem`.

Reply:
I agree having them defined in `FileSystem` is a bit obfuscated. @yuzhu @apc999 wdyt? Maybe we make a separate class `alluxio.hadoop.ConfigurationProperties` to hold the Alluxio-specific HDFS properties?

Reply:
@rongrong Apologies for the late response, I think in lieu of feedback from @yuzhu or @apc999 you can use your own discretion here. wdyt of making an `alluxio.hadoop.ConfigurationProperties` class like suggested above?